tests/test_layer.py

```diff
@@ -214,15 +180,16 @@ def helper_test_add_potting_region(layer):
                     "potting_units": "in",
                     "thickness": 0.1,
                     "standoff": 0.2,
-                    "shape": {"shape_type": "POLYGONAL", "points": [(1, 2), (4.4, 5.5), "INVALID"]},
+                    "shape": shape,
                 },
             ],
         )
-        assert False
+        pytest.fail("No exception thrown when polygonal points list element is incorrect type.")
     except SherlockAddPottingRegionError as e:
         assert str(e) == "Add potting region error: Point 2 invalid for potting region 0."
 
     try:
+        shape = PolygonalShape(points=[(1, 2), (4.4, 5.5, 10), (1, 6)], rotation=123.4)
```

Comment:
Extract the 2nd point value to a variable. E.g. `invalid_point`. This makes it easier to read and understand why this is supposed to fail.